src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java

```diff
@@ -68,17 +68,17 @@ public void testNestedTooDeep() throws Exception {
      *     interface-requirements for test-methods.
      */
     @Test
-    public void testNestedOk() throws Exception {
+    public void testNestedForDepthLevelFour() throws Exception {
 
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
 
         verifyWithInlineConfigParser(
-                getPath("InputNestedForDepth1.java"),
+                getPath("InputNestedForDepthLevelFour.java"),
                expected);
     }
 
     @Test
-    public void testTokensNotNull() {
+    public void testNestedForDepthTokensNotNull() {
```

Comment:
Please revert this for now. We are only changing test methods that extract java input files in this issue